### ISSUE TITLE
Issues with armv7l

### ISSUE THREAD

**Issue by blackliner** [sjmayotte/route53-dynamic-dns#22]
Hi, very much love this app!

I just tried to get it running on my raspi (32bit, otherwise ZFS won't work 😞 ) and got the following errors (both when using your prebuilt docker image with armv7 and `docker build .`)
```
...
Step 9/14 : RUN npm install
 ---> Running in 5f41f199dfa9


#
# Fatal error in , line 0
# unreachable code
#
#
#
#FailureMessage Object: 0xbed8962c
```

Might totally not be related to this app, I will try to figure out whats going on.

**Comment by blackliner** [comment]
the latest alpine based node image (`17.3-alpine`) did not work either. I tried it with `node:17`, which is now giving me this:
```
...
Step 8/14 : COPY package.json /usr/src/app/
 ---> b37a806fedf8
Step 9/14 : RUN npm install
 ---> Running in 5a8615ac3e55
Node.js[7]: ../src/util.cc:188:double node::GetCurrentTimeInMicroseconds(): Assertion `(0) == (uv_gettimeofday(&tv))' failed.
Aborted (core dumped)
The command '/bin/sh -c npm install' returned a non-zero code: 134
```

**Comment by blackliner** [comment]
I tried to replicate it on my x86 machine, looking for an architecture/platform related issue, but it worked (hoped so). So it might be something relevant to the raspi 4B itself.
```bash
sudo apt-get update
sudo apt-get install -y qemu qemu-user-static binfmt-support
docker run --rm --privileged multiarch/qemu-user-static --reset -p yes

git clone git@github.com:sjmayotte/route53-dynamic-dns.git && cd route53-dynamic-dns
docker buildx build --platform linux/arm/v7 .
```

**Comment by blackliner** [comment]
It works with `node:16` as base image 🤷

**Comment by sjmayotte** [comment]
@blackliner do you have any idea why it doesn't work with new node image on that device?  I'm hesitant to go to node 16.  It would be good to understand if there is a known issue that is planned to be fixed.

**Comment by blackliner** [comment]
Not really, all I did was to try out a few permutations of version combinations, and as soon as it worked, I just used it. I did not do any further debugging, as I have 0 experience with npn/NodeJS.

Is it possible to replicate this issue on your end? Given you have access to a Raspi4B

**Comment by ismarslomic** [comment]
@blackliner could you please check if you experience same issue with latest image version` v1.3.0`? It has been updated to use `node:18.14.0-alpine3.17` as base image.
